models/reviewModel.js

```diff
@@ -1,15 +1,31 @@
 const db = require('../db/connection')
 const {createRef} = require('../db/seeds/utils')
 
-exports.fetchReviews = () => {
-    return db.query(`
+exports.fetchReviews = (sort_by = 'created_at', order = 'DESC', category) => {
+    const validSorts = ['review_id', 'title', 'category', 'designer', 'owner', 'review_body', 'review_img_url', 'create_at', 'votes']
+    const queries = []
+    
+    if (validSorts.indexOf(sort_by) === -1) sort_by = 'created_at'
```

Comment:
I would rather see you reject this. From a client stand point, it would make more sense to be told that i've made an error as opposed to just get something different from my request. Same for your asc/desc